modules/storage-persistent-storage-azure-volume-format.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage-azure.adoc
+
+[id="volume-format-azure_{context}"]
+= Volume Format
```

Comment:
Should this be sentence case, or is "Volume Format" a thing?

Reply:
Adjusted to sentence case.